src/components/ProjectCard/index.jsx

```diff
@@ -53,7 +57,9 @@ export default class ProjectCard extends Component {
               {name}
```

Comment:
Changing the variant of the first Typography from `title` to `headline` seems to sit better in the UI. Let's do that :)